### PR TITLE
Fix some typo's and improve some language constructs

### PR DESCRIPTION
**Pull request by WouterSioen**

### CODE REVIEW

**Review by rathesDot — DISMISSED** [review]
#hacktoberfest 😂

**Comment by WouterSioen** [comment]
@rathesDot Some of those typo's were already bothering for quite a while 😉

File: apiary.apib [hunk 1]
```diff
@@ -85,7 +85,7 @@ To ensure a fast and predictable experience for everyone, all our API endpoints
 
 When you go over this limit, our API will respond with a `HTTP 429 Too Many Requests error`.
 
-If your integrations needs a higher limit, send us an email at [api@teamleader.eu](mailto:api@teamleader.eu) with a detailed explanation why you need an increased rate limit.
+If your integrations needs a higher limit, send us an email at [api@teamleader.eu](mailto:api@teamleader.eu) with a detailed explanation of why you need an increased rate limit.
```
Comment (kevinwaelkens) [inline apiary.apib]: `If your integrations neeD a higher limit`.

Comment (WouterSioen) [reply apiary.apib]: "your integration needs" seems correct to me? 🤔

Comment (kevinwaelkens) [reply apiary.apib]: That is correct, but it says `your integrationS` 🙈 	so then it needs to be `needS`

Comment (WouterSioen) [reply apiary.apib]: oh yeah, I've read over the `s` in integrations. That makes it completely different, well spotted 🔍

**Review by kevinwaelkens — APPROVED** [review]

**Review by rathesDot — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -206,9 +206,9 @@ The required POST parameters are:
...
+### Client-side authorization (implicit grant)
 
-Your client secret must remain private and can not be embedded within client side applications such as browser plugins. For these types of applications we have an alternative authorization method called using the [implicit grant](https://tools.ietf.org/html/rfc6749#section-1.3.2). The implicit grant is a simplified authorization code flow optimized for clients implemented in a browser using a scripting language such as JavaScript. In the implicit flow, instead of issuing the client an authorization code, the client is issued an access token directly.
+Your client secret must remain private and can not be embedded within client side applications such as browser plugins. For these types of applications, we have an alternative authorization method called using the [implicit grant](https://tools.ietf.org/html/rfc6749#section-1.3.2). The implicit grant is a simplified authorization code flow optimized for clients implemented in a browser using a scripting language such as JavaScript. In the implicit flow, instead of issuing the client an authorization code, the client is issued an access token directly.
```
Comment (dietrichm) [inline apiary.apib:211]: PP: "within client-side applications" (dash between client and side).

Comment (kevinwaelkens) [reply apiary.apib:211]: fyi, it's called a hyphen in this case @dietrichm

**Review by JordyDeGroeve — APPROVED** [review]

**Review by paultimoce — APPROVED** [review]
looks good

**Comment by dietrichm** [comment]
But... but... what about that missing *hyphen*?

**Pull request merged**